daisy/workflow/step_delete_resources.go

```diff
@@ -56,7 +57,10 @@ func (d *DeleteResources) run(w *Workflow) error {
 		wg.Add(1)
 		go func(i string) {
 			defer wg.Done()
-			if err := w.deleteInstance(namer(i, w.Name, w.id)); err != nil {
+			r, ok := w.instanceRefs.get(i)
```

Comment:
I know we didn't allow it before but should we check isLink in DeleteResources? It will complicate the logic some.